demo/src/layout/header-component.js

```diff
@@ -0,0 +1,41 @@
+import githubLogoSvg from 'bundle-text:../../img/github-logo.svg';
+import srgssrLogo from '../../img/srgssr-logo.png';
+import Pillarbox from '../../../src/pillarbox';
+import { html, LitElement, unsafeCSS } from 'lit';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+import '../router/route-link-component';
+import { theme } from '../theme/theme';
+import headerCSS from 'bundle-text:./header-component.scss';
+
+export class HeaderElement extends LitElement {
+  static styles = [theme, unsafeCSS(headerCSS)];
+
+  constructor() {
+    super();
+  }
```

Comment:
Please refer to https://github.com/SRGSSR/pillarbox-web/pull/141/files#r1429904135

Reply:
Removed